com.github.Ryujinx.json

```diff
@@ -0,0 +1,1266 @@
+{
+    "app-id": "com.github.Ryujinx",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.dotnet"],
+    "command": "start",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host"
+    ],
+    "modules": [
+	{
+            "name": "start",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -d /app/bin",
+  "              install -Dm555 start /app/bin"
+	    ],
+            "sources": [
+		{
+                    "type": "script",
+                    "dest-filename": "start",
+                    "commands": [ 
+			"DOTNET_ROOT=/app/dotnet/lib/ /app/ryujinx/Ryujinx"
+		    ]
+	        }
+	    ]
+	},
+	{
+	    "name": "dotnet",
+	    "buildsystem": "simple",
+	    "build-commands": [
+                "/usr/lib/sdk/dotnet/install.sh"
+	    ]
+	},
+        {
+            "name": "ryujinx",
+            "buildsystem": "simple",
+            "build-commands": [
+		"mkdir /app/ryujinx",
+		". /usr/lib/sdk/dotnet/enable.sh; dotnet publish -c Release --source /run/build/ryujinx/nuget-sources --output /app/ryujinx" ,
+		"ln -s /app/ryujinx/Ryujinx /app/bin"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/Ryujinx/Ryujinx",
+		    "commit": "09341dc11d88de39b2b4fb990ec730b4d57f8b0a"
+                },
+                {
+                    "type": "patch",
+                    "path": "remove-test.patch"
+                },
+                {
```

Comment:
Any chance you could move generated part to `generated-sources.json`?

Reply:
Much cleaner!